src/messages/formatters.ts

```diff
@@ -0,0 +1,154 @@
+/**
+ * Wraps the content inside a codeblock with no language.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function codeBlock<C extends string>(content: C): `\`\`\`\n${C}\`\`\``;
+/**
+ * Wraps the content inside a codeblock with the specified language.
+ * @param language The language for the codeblock.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function codeBlock<L extends string, C extends string>(language: L, content: C): `\`\`\`${L}\n${C}\`\`\``;
+export function codeBlock(language: string, content?: string): string {
+  return typeof content === 'undefined' ? `\`\`\`\n${language}\`\`\`` : `\`\`\`${language}\n${content}\`\`\``;
+}
+
+/**
+ * Wraps the content inside an inline code.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function inlineCode<C extends string>(content: C): `\`${C}\`` {
+  return `\`${content}\``;
+}
+
+/**
+ * Formats the content into italic text.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function italic<C extends string>(content: C): `*${C}*` {
+  return `*${content}*`;
```

Comment:
```suggestion
  return `_${content}_`;
```

Single underscores are the more common across platforms form of _italic_ afaik